conda/psychopy-env.yml

```diff
@@ -8,5 +8,5 @@ dependencies:
 - pip:
   - psychtoolbox
   - pygame
-  - pyparallel
-  - SoundFile
+  - pyparallel; platform_system == "Darwin" or platform_system == "Linux"
```

Comment:
matter of taste so I'll leave this up to you, but I believe `!= Windows` would work too and would be more concise.

Reply:
Yes, this is better. Changed.